src/main/solution_space/solution_space_handler.py

```diff
@@ -54,17 +63,30 @@ def __find_same_fragments(solutions: pd.DataFrame, start_index: int) -> Tuple[in
     return i, ati_elements
 
 
-def __get_profile(solutions: pd.DataFrame, index: int) -> Profile:
-    age = __get_column_value(solutions, index, consts.CODE_TRACKER_COLUMN.AGE.value)
-    experience = __get_column_value(solutions, index, consts.CODE_TRACKER_COLUMN.EXPERIENCE.value)
+def __get_experience_from_str(experience: str) -> Union[EXPERIENCE, DEFAULT_VALUES]:
+    for e in EXPERIENCE:
+        if e.value == experience:
+            return e
+    return DEFAULT_VALUES.EXPERIENCE
```

Comment:
You forgot about `value` attribute: `return DEFAULT_VALUES.EXPERIENCE.value` and  `return e.value` too